### PR TITLE
Deduplicate tags for spans

### PR DESCRIPTION
**Pull request by rubenvp8510**
Signed-off-by: Ruben Vargas <ruben.vp8510@gmail.com>

## Which problem is this PR solving?
- This solves issue: https://github.com/jaegertracing/jaeger/issues/1470, at least from the UI perspective, not sure if this is the best way.

## Short description of the changes
- I check for duplicated tags, and remove it (I left only one of them)
- Also has a boolean to indicate that span has duplicated tags so we can show a warning or something 

Screenshot of warning:

Screenshot from 2019-05-03 12-29-02

### CODE REVIEW

**Comment by yurishkuro** [comment]
I like the warning icon, but we have a built-in place for warnings in the spans (below), which may contain more than one warning. So it would be better to display them as a list. Maybe we can add another expandable item below logs called `Warnings({count})`, which may be prefixed with the warning icon for better visuals, and not show up at all if span has no warnings.

https://github.com/jaegertracing/jaeger/blob/3fff10206109b54c9d8a42cb1347336836a9b0ea/model/model.pb.go#L359

**Comment by rubenvp8510** [comment]
Ahh! I haven't noticed that field in span object, in that case it makes more sense to put warnings there. I'll do the modifications and update this PR, with new screenshots also.

**Comment by rubenvp8510** [comment]
Screenshot from 2019-05-04 13-09-57

@yurishkuro WDYT?

**Comment by yurishkuro** [comment]
@rubenvp8510 s/s looks good to me. I  assume the Warnings will not show up if there are no warnings?

@tiffon is this ready to merge?

**Comment by rubenvp8510** [comment]
@yurishkuro yes, the warning accordion  does not show up if there are no warning on the span

File: packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/SpanDetail/AccordianText.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  cursor: initial;
+}
+
+.AccordianText--header.is-high-contrast:not(.is-empty):hover {
+  background: #ddd;
+}
+
+.AccordianText--emptyIcon {
+  color: #aaa;
+}
```
Comment (tiffon) [inline packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/SpanDetail/AccordianText.css]: I don't think these styles are used anywhere?

- `.AccordianText--header.is-empty`
- `.AccordianText--header.is-high-contrast:not(.is-empty):hover` (as a result of the above)
- `.AccordianText--emptyIcon`

**Comment by rubenvp8510** [comment]
Hi @tiffon I did all requested modifications, also I modified the style according to your suggestion, I think looks better and more "inline" with the actual UI style.

I attached some screen-shots of the latest modifications.

Screenshot from 2019-05-15 18-23-30
Screenshot from 2019-05-15 18-23-19

**Comment by rubenvp8510** [comment]
Looks very good! Thanks for the improvement!

**Comment by tiffon** [comment]
Hey, awesome, done! :+1:

**Pull request merged**